site/en/guide/distributed_training.ipynb

```diff
@@ -503,14 +503,14 @@
       "source": [
         "## Use tf.distribute.Strategy with Keras Model.fit\n",
         "\n",
-        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](/keras/customizing_what_happens_in_fit.ipynb).\n",
+        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io/api/). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](keras-io/guides/ipynb/customizing_what_happens_in_fit.ipynb).\n",
```

Comment:
Relative linking (https://www.tensorflow.org/community/contribute/docs_style#links_in_markdown_and_notebooks)

Let's try the following:

```suggestion
        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io/api/). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](./keras/customizing_what_happens_in_fit.ipynb).\n",
```